resources/views/group_create.blade.php

```diff
@@ -0,0 +1,47 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container mt-4">
+        <form id="create-form" method="post" action="{{ route('groups.store') }}">
+            @csrf
+            <div class="form-group row">
+                <label for="name" class="pl-5 col-md-2 col-form-label text-md-left">
+                    <div class="h5">
+                        {{ trans('name') }}
+                    </div>
+                </label>
+                <div class="col-md">
+                    <input id="name" type="text" class="form-control @error('name') is-invalid @enderror" name="name" autocomplete="name" autofocus>
+                    @error('name')
```

Comment:
```suggestion
                    @error ('name')
```
sửa hết lỗi này bên dưới nếu có